source/docs/tag-plugins/wavedrom.md

```diff
@@ -0,0 +1,52 @@
+---
+title: WaveDrom
+description: NexT User Docs – NexT Supported Tags – WaveDrom
+---
+
+### Settings
+
+```yml NexT config file
+# WaveDrom tag
+wavedrom:
+  enable: true
+```
+
+```yml Hexo config file
+highlight:
+  exclude_languages:
+    - wavedrom
+```
+
+### Usage
+
+```jinja
+{% wavedrom %}
+{% endwavedrom %}
+```
+
+or
+
+~~~markdown
+```wavedrom
```

Comment:
It seems that wavedrom does not support this syntax

Reply:
Ouch! You are right! I debugged a bit more. Although the hexo tag plugin doc mentioned the [backtick code block](https://hexo.io/docs/tag-plugins#Backtick-Code-Block). But the rendering doesn't really process them with tag extends. Only the swigs will be processed. Let me update the docs.

Reply:
doc is updated now.

Reply:
PR description is also updated.